### ISSUE TITLE
Show how models (Meshes) can be loaded dynamically

### ISSUE THREAD

**Issue by ADefWebserver** [ADefWebserver/BlazorBabylonJs#2]
Make a new page that has possible models (meshes) in a dropdown. Selecting one will load it.

**Comment by SQL-MisterMagoo** [comment]
PR #4 introduces a loadMesh function, but it still has some stuff that is pretty specific to that particular file I think. In reality we might need some way to allow customisation code which is specific to a model

**Comment by ADefWebserver** [comment]
I agree. I am looking at what you did and "learning" how it all fits together. 

I have to laugh because the direction I was going was waaaay off from where this project needs to go to be truly useful.
